.circleci/config.yml

```diff
@@ -1,38 +1,44 @@
 version: 2.1
 
 orbs:
-  saucectl: friggahel/saucectl@<<pipeline.parameters.dev-orb-version>>
+  saucectl: saucelabs/saucectl-run@<<pipeline.parameters.dev-orb-version>>
   orb-tools: circleci/orb-tools@10.0
   shellcheck: circleci/shellcheck@2.0
 
-# Pipeline Parameters
-## These parameters are used internally by orb-tools. Skip to the Jobs section.
 parameters:
   run-integration-tests:
     description: An internal flag to prevent integration test from running before a development version has been created.
     type: boolean
     default: false
   dev-orb-version:
-    description: >
+    description: |
       The development version of the orb to test.
       This value is automatically adjusted by the "trigger-integration-tests-workflow" job to correspond with the specific version created by the commit and should not be edited.
       A "dev:alpha" version must exist for the initial pipeline run.
     type: string
     default: "dev:alpha"
 
 jobs:
-  # Define one or more jobs which will utilize your orb's commands and parameters to validate your changes.
-  integration-test-1:
+  integration-test_saucectl:
     docker:
       - image: cimg/base:stable
     steps:
       - checkout
-      - saucectl/install
-      - saucectl/run
+      - setup_remote_docker:
+          version: 20.10.2
+      - saucectl/saucectl-run:
+          show-console-log: true
+          config-file: ./.sauce/config.yml
+          region: us-west-1
+          working-directory: ./tests/
+      - saucectl/saucectl-run:
```

Comment:
This is defined twice? 🤔

Reply:
Actually this is not a definition. This is a usage one is to run against docker, one to the cloud. 

It mainly mean use `saucectl-run` from orb `saucectl` and configuration is the nested object.